SignalServiceKit/src/Loki/API/FileServerAPI.swift

```diff
@@ -52,6 +53,37 @@ public final class FileServerAPI : DotNetAPI {
         }
     }
     
+    @objc(downloadProfilePicture:)
+    public static func objc_downloadProfilePicture(_ downloadURL: String) -> AnyPromise {
+        return AnyPromise.from(downloadAttachment(downloadURL))
+    }
+    
+    // MARK: Attachment Download
+    @objc(downloadAttachment:)
+    public static func objc_downloadAttachment(_ downloadURL: String) -> AnyPromise {
+        return AnyPromise.from(downloadAttachment(downloadURL))
+    }
+    
+    public static func downloadAttachment(_ downloadURL: String) -> Promise<Data> {
+        var error: NSError?
+        var url = downloadURL
+        if downloadURL.contains(fileStaticServer) {
+            url = downloadURL.replacingOccurrences(of: fileStaticServer, with: "\(server)/loki/v1")
```

Comment:
`replacingOccurrences(of:with:)` is a no-op if `downloadURL` doesn't contain `fileStaticServer`, so the `contains(_:)` check isn't necessary :)